sdk/eventgrid/azure-eventgrid/azure/eventgrid/_consumer.py

```diff
@@ -57,11 +60,8 @@ def deserialize_eventgrid_events(self, eventgrid_event, **kwargs): # pylint: dis
         try:
             eventgrid_event = EventGridEvent._from_json(eventgrid_event, encode) # pylint: disable=protected-access
```

Comment:
same as above, will need updating once `_from_json` is changed `_to_json`

Reply:
I have this tracked in the decode changes where i'm including a _to_json as well (along with a _from_json) https://github.com/Azure/azure-sdk-for-python/issues/16352